### PR TITLE
demo game model

### PR DESCRIPTION
**Pull request by artemderkach**
just a DEMO version.
no validation,
no tests (sorry Glib)

### CODE REVIEW

**Comment by Fleischers** [comment]
Honestly, I even can't imagine how to use it. Will take time to investigate

**Comment by artemderkach** [comment]
there is small description in the end of game file.

**Comment by artemderkach** [comment]
descriptioon: https://github.com/GorlifSense/rocket-snake/wiki

File: models/Snake.js [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
   addPointTail(point) {
-    this.body.push(point);
+    return this.body.push(point);
   }
 
   removePointTail() {
     return this.body.pop();
   }
 
   // check if input direction is valid depends on snake position
```
Comment (Belrestro) [inline models/Snake.js:23]: The one direction snake can't move is opposite direction

File: models/Game.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+const Grid = require('./Grid.js');
+const Snake = require('./Snake.js');
+const Point = require('./Point.js');
+
+const GRID_HEIGHT = 50;
+const GRID_WIDTH = 50;
+const SNAKE_LENGTH = 3;
```
Comment (Belrestro) [inline models/Game.js:9]: Initial length ?

**Review by Belrestro — APPROVED** [review]
Great job

**Pull request merged**